plan-patches/cf-azure/README.md

```diff
@@ -0,0 +1,77 @@
+# Patch: cf-azure
+
+Steps to deploy cf with bbl:
+
+1. Follow the normal steps to bbl up with a patch.
+
+    ```bash
+    export BOSH_BOOTLOADER=<YOUR BOSH BOOTLOAER PATH>
```

Comment:
Do you think we can fix this small typo? BOOTLOAER -> BOOTLOADER

Reply:
changed. thanks.